.github/workflows/Windows-pack.yml

```diff
@@ -0,0 +1,131 @@
+name: Packaging(Windows)
+
+on:
+  push:
+    branches: [ master ]
+    paths-ignore:
+      - 'README.md'
+      - 'LICENSE'
+  pull_request:
+    branches: [ master ]
+    paths-ignore:
+      - 'README.md'
+      - 'LICENSE'
+
+env:
+  PRODUCT: flameshot
+  VERSION: 0.8.0
```

Comment:
Shouldn't this be dynamic (e.g read from "${{ github.workspace }}/CMakeLists.txt")?